forest/src/cli/config.rs

```diff
@@ -34,9 +34,15 @@ pub struct Config {
 
 impl Default for Config {
     fn default() -> Self {
+        let dir = ProjectDirs::from("com", "ChainSafe", "Forest").expect("failed to find project directories, please set FOREST_CONFIG_PATH environment variable manually.");
         Self {
             network: Libp2pConfig::default(),
-            data_dir: get_home_dir() + "/.forest",
+            data_dir: dir
```

Comment:
In general, why don't we want to hold the `data_dir` in a `PathBuf`? Seems more useful and safer to have properly typed paths. Not a blocker for me, just a suggestion.

Reply:
Yes, `data_dir` should be a `PathBuf`. The toml parsing still works fine on PathBufs.